x-pack/filebeat/module/coredns/kubernetes/test/coredns-json.log-expected.json

```diff
@@ -0,0 +1,44 @@
+[
+    {
+        "@timestamp": "2019-02-12T00:27:28.903Z",
+        "coredns.dnssec_ok": "false",
+        "coredns.id": "21583",
+        "coredns.logging_level": "INFO",
+        "coredns.query.class": "IN",
+        "coredns.query.name": "httpbin.org.cluster.local.",
+        "coredns.query.size": 43,
+        "coredns.query.type": "A",
+        "coredns.response.code": "NXDOMAIN",
+        "coredns.response.flags": [
+            "QR",
+            "RD",
+            "RA"
+        ],
+        "coredns.response.size": 136,
+        "ecs.version": "1.0.0-beta2",
+        "event.dataset": "coredns.kubernetes",
+        "event.duration": 102078,
+        "event.module": "coredns",
+        "fileset.name": "kubernetes",
+        "input.type": "log",
+        "kubernetes.container.name": "coredns",
+        "kubernetes.labels.k8s-app": "kube-dns",
+        "kubernetes.labels.pod-template-hash": "86c58d9df4",
+        "kubernetes.namespace": "kube-system",
+        "kubernetes.node.name": "minikube",
+        "kubernetes.pod.name": "coredns-86c58d9df4-jwhsg",
+        "kubernetes.pod.uid": "d57d545e-2a9d-11e9-995f-08002730e0dc",
+        "kubernetes.replicaset.name": "coredns-86c58d9df4",
+        "log.offset": 0,
+        "message": "2019-02-12T00:27:28.903Z [INFO] 172.17.0.4:36413 - 21583 \"A IN httpbin.org.cluster.local. udp 43 false 512\" NXDOMAIN qr,rd,ra 136 0.000102078s",
+        "network.transport": "udp",
+        "service.type": "coredns",
+        "source.address": "172.17.0.4",
+        "source.ip": "172.17.0.4",
+        "source.port": "36413",
+        "stream": "stdout",
```

Comment:
I'm worried this might conflict in the future having a keyword on the top level. @webmat Some suggestions where to put this?

Reply:
you mean "stream" field? I can just drop it.

Reply:
SGTM.